js/aboutApp/about-app.css

```diff
@@ -0,0 +1,39 @@
+html, body {
+    margin: 0;
+    height: 100%;
+    font-family: sans-serif;
+}
+
+.content {
+    left: 50%;
+    position: absolute;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    margin: 0 auto;
+    text-align: center;
+}
+
+.logo {
+    width: 100px;
+    height: 100px;
+}
+
+.name {
+    display: block;
+    font-size: 1.3em;
```

Comment:
Please verify this with the design team. This may be slightly larger than what we currently have for the Mac Wrapper.